app/components/auk/dropdown/item.hbs

```diff
@@ -0,0 +1,27 @@
+<li class={{@class}}>
+  {{#if @href}}
+    <a class="auk-menu__item {{this.skin}}" href={{@href}} ...attributes>
+      {{yield}}
+    </a>
+  {{else if @textOnly}}
+    <div class="auk-menu__item--text-only" ...attributes>
+      {{yield}}
+    </div>
+  {{else}}
+    {{!-- TODO: when wanting to define an action in a dropdown that looks (and is) clickable, but doesn't link anywhere, one ends up here.
```

Comment:
Dat is een discussie die eigenlijk al jaren aan het lopen is, maar geen definitieve conclusie uit komt (wegens historiek, context, etc.). Nu, wat meestal in de kantlijn overééngekomen wordt is dat:
- buttons beter als individueel element gebruikt worden binnen een form-gerelateerde context, tenzij ze (om functionaliteit of accessibility te bewaren) custom styling krijgen (zoals bij autocomplete, file-upload, etc. add-ons).
- a-tags beter dienen voor integratie binnen andere componenten.

Maar, om duidelijk te zijn, dit natuurlijk geen gouden regel en er zijn altijd uitzonderingen waarbij het ene beter dan de andere is.

Nu, ik zou dit zo laten tenzij er echt een goede reden voor een switch zou zijn, maar discussies mbt semantics zijn altijd belangrijk en welkom 👍